playwright-tests/tests/theia-app.test.ts

```diff
@@ -0,0 +1,67 @@
+/********************************************************************************
+ * Copyright (C) 2022 EclipseSource and others.
```

Comment:
Shouldn't be the copyright Ericsson and others? I think you wrote this file.

2024?

Reply:
In the latest version I renamed that file to "trace-viewer-app.test.ts". The file now has two headers - the original one from the corresponding file from `theia-playwright-template` and a second one, for our changes. The second one has copyright year 2024.